lib/routes/stbu/xyxw.ts

```diff
@@ -9,19 +9,31 @@ import timezone from '@/utils/timezone';
 const gbk2utf8 = (s) => iconv.decode(s, 'gbk');
 export const route: Route = {
     path: '/xyxw',
+    categories: ['university'],
+    example: '/stbu/xyxw',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: true,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
     radar: [
         {
-            source: ['stbu.edu.cn/html/news/xueyuan', 'stbu.edu.cn/'],
+            source: ['stbu.edu.cn/html/news/xueyuan', 'stbu.edu.cn'],
+            target: '/stbu/xyxw',
```

Comment:
If `target` is not specified, it will be the same as `path`, and the Radar output will be `/stbu/xyxw` since the namespace will be automatically prepended during build. However, if `target` is set to `/stbu/xyxw`, the Radar output will be `/stbu/stbu/xyxw`.